### PR TITLE
Migrated champion mastery endpoint to v3

### LINKED ISSUE #348: Move the champion mastery API to V3
API ref: https://developer.riotgames.com/api-methods/#champion-mastery-v3
Code ref: https://github.com/BenFradet/RiotSharp/blob/develop/RiotSharp/RiotApi.cs#L1221-L1350

### PR DESCRIPTION
**Pull request by JanOuborny**
Fixes #348 

TODO:
~~Remove `usePlatform` parameter from `RateLimitedRequester`~~

### CODE REVIEW

File: RiotSharp/RiotApi.cs [hunk 1]
```diff
@@ -562,80 +557,79 @@ public FeaturedGames GetFeaturedGames(Region region)
...
+            var requestUrl = string.Format(ChampionMasteriesBySummonerUrl, summonerId);
 
-            var json = requester.CreateGetRequest(rootUrl + ChampionMasteryAllChampions,
-                platform.ConvertToRegion());
+            var json = requester.CreateGetRequest(ChampionMasteryRootUrl + requestUrl, region, usePlatforms: true);
             return JsonConvert.DeserializeObject<List<ChampionMastery>>(json);
         }
-      
-        public async Task<List<ChampionMastery>> GetChampionMasteriesAsync(Platform platform, long summonerId)
+
```
Comment (BenFradet) [inline RiotSharp/RiotApi.cs]: is there a white space there?

File: RiotSharp/RiotApi.cs [hunk 2]
```diff
@@ -562,80 +557,79 @@ public FeaturedGames GetFeaturedGames(Region region)
...
 
-            var json = await requester.CreateGetRequestAsync(rootUrl + ChampionMasteryAllChampions,
-                platform.ConvertToRegion());
+            var json = await requester.CreateGetRequestAsync(ChampionMasteryRootUrl + requestUrl, 
+                region, usePlatforms: true);
             return (await Task.Factory.StartNew(() => JsonConvert.DeserializeObject<List<ChampionMastery>>(json)));
-        }
-    
-        public int GetTotalChampionMasteryScore(Platform platform, long summonerId)
+        }
```
Comment (BenFradet) [inline RiotSharp/RiotApi.cs]: here too?

File: RiotSharp/RiotApi.cs (hunk 1 above)
Comment (JanOuborny) [inline RiotSharp/RiotApi.cs]: Yep, it is.

**Comment by JanOuborny** [comment]
@BenFradet Can't run unit tests right now, because Riot's server are broken.

**Pull request merged**